workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.ts

```diff
@@ -29,45 +29,59 @@ interface TokensExpiration {
   refreshToken: number; // ms
 }
 
-export interface CognitoAuthenticationPluginOptions {
+interface CognitoAppClient {
   /**
-   * The Cognito domain. Follows the format: "https://\<domain prefix\>.auth.\<region\>.amazoncognito.com"
+   * The Cognito app client IDs.
    */
-  cognitoDomain: string;
-
+  clientId: string | string[];
```

Comment:
Each AppClient should only have one client ID associated with it

Reply:
They are referring to allowing for multiple app clients from the same user pool. The current implementation only allows for a single app client (the web UI app client)